src/components/User/DataList/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { StyledTable, StyledTableTh, StyledTableTd } from "./styles";
+
+const DataList = ({ data }) => {
+  return (
+    <StyledTable>
+      <thead>
+        <tr>
+          <StyledTableTh>First Name</StyledTableTh>
+          <StyledTableTh>Last Name</StyledTableTh>
+          <StyledTableTh>Email</StyledTableTh>
+          <StyledTableTh>City</StyledTableTh>
+        </tr>
+      </thead>
+      <tbody>
+        {data.map((i) => (
```

Comment:
Add data checking against undefined, since it can be undefined if api response is not received yet